lib/init.coffee

```diff
@@ -6,13 +6,13 @@ module.exports =
     executablePath:
       title: 'Executable Path'
       type: 'string'
-      default: 'pep257'
-      description: "Path to executable pep257 cmd."
+      default: 'pydocstyle'
```

Comment:
Great! 

Whatever solution you implement should also probably be copied over to https://github.com/AtomLinter/linter-pep8/issues/55 since that project is also undergoing a name change.